docs/usage.md

```diff
@@ -78,6 +78,10 @@ To prevent entire pipeline failures to due a single 'bad sample', nf-core/funcsc
 
 > ⚠️ If a sample does not reach this contig length threshold, you will recieve a warning in your console and `.nextflow.log` file, and no result files will exist for this sample in your results directory.
 
+We also noticed that when annotation is run with Prokka, the `.gbk` file passed to AntiSMASH may produce the error _'translation longer than location allows'_ and end the pipeline run.
+
+> ⚠️ If AntiSMASH is run for BGC detection it is recommended NOT to run PROKKA for annotation but instead leave the default annotation tool Prodigal or use BAKTA.
```

Comment:
When the annotation is run with Prokka, its `.gbk` file of a sample passed to antiSMASH may produce the error _'translation longer than location allows'_ and end the pipeline run. This Prokka bug was recognized before (see [here](https://github.com/antismash/antismash/discussions/450)) and is not likely to be fixed soon.

> ⚠️ If antiSMASH is run for BGC detection it is recommended to not run Prokka for annotation but instead leave the default annotation tool Prodigal or switch to Bakta.
```

Reply:
Include caveat that bakta is bacteria only